src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/MessageboxInstancesController.cs

```diff
@@ -442,5 +451,47 @@ private async Task<StringValues> MatchStringToAppTitle(string searchString)
 
             return new StringValues(appIds.ToArray());
         }
+
+        private void GetStatusFromQueryParams(
+           bool includeActive,
+           bool includeArchived,
+           bool includeDeleted,
+           Dictionary<string, StringValues> queryParams)
```

Comment:
Could this method return a new dictionary instead? And then the caller can append it to the queryParams dictionary. A little easier to read.